server/src/com/thoughtworks/go/server/controller/AgentRegistrationController.java

```diff
@@ -276,14 +284,10 @@ private void sendFile(InputStreamSrc input, HttpServletResponse response) throws
         }
     }
 
-    private boolean partialElasticAgentAutoregistrationInfo(String elasticAgentId, String elasticPluginId) {
+    private boolean partialElasticAgentInfo(String elasticAgentId, String elasticPluginId) {
```

Comment:
Move this to `AgentConfig`?

Reply:
AgentConfig is already having `isElastic`. I will check and update it.